Sources/Extensions/UIKit/UISearchBarExtensions.swift

```diff
@@ -9,7 +9,7 @@
 #if canImport(UIKit)
 import UIKit
 
-#if !os(watchOS)
+#if os(iOS)
```

Comment:
@omaralbeik  UISearchBar is also available on tvOS SDK, any reason to make it just iOS?

Reply:
It looks like it is supported, but you can not create an instance of it for some reason, I couldn't run tests on tvOS due to the following errors
```
- 'init()' is unavailable
- 'init(frame:)' is unavailable
```
https://forums.developer.apple.com/thread/21421